tests/integration/test_workflows.py

```diff
@@ -17,7 +17,7 @@
 
 FIXTURES_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "fixtures")
 FILES_FIXTURES_PATH = os.path.join(FIXTURES_PATH, "files.json")
-ENV_FIXTURE_PATH = os.path.join(FIXTURES_PATH, "env.yml")
+ENV_FIXTURE_PATH = os.path.join(FIXTURES_PATH, "env.example.yml")
```

Comment:
I believe we might want to keep this as `env.yml` that way people can just create/update their env.yml file instead of having to be cautious about not committing their env.example.yml file